docs/api-inference/hub-integration.md

```diff
@@ -0,0 +1,17 @@
+# Hub Integration
+
+**TODO:**
+- search for models (UI): https://huggingface.co/models?inference_provider=fireworks-ai&sort=trending
+- search all models (UI): https://huggingface.co/models?inference_provider=all&sort=trending
+- set custom API key from provider (optional)
+- order providers by preference (apply to widget and code snippets)
+
+### Features using Inference Providers
+
+Several Hugging Face features utilize the Inference Providers API and count towards your monthly credits:
+
+- Inference Widgets: Interactive widgets available on model pages.
+- [Inference Playground](https://huggingface.co/playground): A comprehensive chat interface supporting various models and providers.
+- Data Studio AI: Converts text to SQL queries for datasets.
```

Comment:
you can "delegate" to e.g. @mishig25, @cfahlgren1 etc to wrap a small blurb about those (with a screenshot)

Reply:
wouldn't mind some help :pray: 

For now I've taken screenshots from the website but can't say it looks so nice: https://moon-ci-docs.huggingface.co/docs/api-inference/pr_1652/en/hub-integration :grimacing: